src/main/java/com/textmessenger/service/LoginServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package com.textmessenger.service;
+
+import com.textmessenger.model.entity.dto.LoginRq;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'com.textmessenger.model.entity.User'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=142&issues=AWWFmk07XKENQn1S911j&open=AWWFmk07XKENQn1S911j)

[ISSUE_KEY]: AWWFmk07XKENQn1S911j